divide_and_conquer/strassens_matrix_mulitplication.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+
+Code for Strassen's Matrix Multiplication Algorithm
+
+Strassen's Matrix Multiplication techinque works upon the divide and conquer methedology.
+
+Here, in the following code:
+1) We have declared variables in accordance to the requirements
+2) We ran two loops for taking input for 2 matrices, which are to be mulitplied
+3) Then we took the elements from the matrices and assign them to variables p to v in accordance to the Strassen's Method which are
+    p = a * (f - h)
+    q = (a + b) * h
+    r = (c + d) * e
+    s = d * (g - e)
+    t = (a + d) * (e + h)
+    u = (b - d) * (g + h)
+    v = (a - c) * (e + f)
+4) Then we assigned the elements of the resultant matrix according to the Strassen's method, which is
+        C = |t+s-q+u    p+q   |
+            |   r+s    p+t-r-v|  
+5) Then, we ran a loop in order to print the resultant matrix
+
+
+Code Contributed by: Tiger Sharks
+
+*/
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int m1[2][2], m2[2][2], res[2][2], x, y;

```

Comment:
To register matrices, you should use C++-style vectors, which could be either `std::vector` (requires the `vector` library) or `std::array` (requires the `array` library). You'll need to create a vector inside a vector.

Reply:
Actually, I am not familiar with style vectors, never used them, and the existing method is the method which I am comfortable with...😅

Reply:
Feel free to come and discuss that in the [Discord](https://the-algorithms.com/discord) server. I could send you a few resources to learn about it. This is the format we use in our repositories.

Reply:
Okay. I refer some resources and do the needful ASAP...

Reply:
I made the required changes and reuploaded the code. Do check it out.